src/components/grid/examples/grid.scss

```diff
@@ -60,35 +60,35 @@ limel-grid {
 // when placing real components into a grid.
 
 my-deep-red-component {
-    background-color: var(--lime-deep-red);
+    background-color: rgb(var(--color-red-default));
 }
 my-red-component {
-    background-color: var(--lime-red);
+    background-color: rgb(var(--color-coral-default));
 }
 my-orange-component {
-    background-color: var(--lime-orange);
+    background-color: rgb(var(--color-orange-default));
 }
 my-yellow-component {
-    background-color: var(--lime-yellow);
+    background-color: rgb(var(--color-yellow-default));
 }
 my-green-component {
-    background-color: var(--lime-green);
+    background-color: rgb(var(--color-green-default));
 }
 my-turquoise-component {
-    background-color: var(--lime-turquoise);
+    background-color: rgb(var(--color-teal-default));
 }
 my-blue-component {
-    background-color: var(--lime-blue);
+    background-color: rgb(var(--color-sky-default));
 }
 my-dark-blue-component {
-    background-color: var(--lime-dark-blue);
+    background-color: rgb(var(--color-blue-default));
 }
 my-magenta-component {
-    background-color: var(--lime-magenta);
+    background-color: rgb(var(--color-magenta-default));
 }
 my-light-grey-component {
-    background-color: var(--lime-light-grey);
+    background-color: rgb(var(--color-glaucous-default));
 }
 my-dark-grey-component {
-    background-color: var(--lime-dark-grey);
+    background-color: rgb(var(--color-gray-default));
```

Comment:
Except for the two blue components, the new colors are easier to distinguish, which is better.

![image](https://user-images.githubusercontent.com/154725/114524099-36bd8a80-9c45-11eb-9fb4-35a70907f746.png)

But, some of the colors no longer match their component names. In this example, it's fairly important that they do, for the reader to be able to follow the code and see how the code produces the end result.

![image](https://user-images.githubusercontent.com/154725/114524361-6ff5fa80-9c45-11eb-96a5-cc1172a1b2bf.png)

Note that both the component name and the short identifier ('red', 'blu', 'mag', etc) needs to match. We could make the identifiers a little longer if we want to, but they should preferably all have the same number of characters, so they line up nicely in a grid matching the finished rendered grid…

(Either of us can fix it, but I'm leaving this as a note regardless to remember it should be done.)

Reply:
Good point. We let each other know if either of us starts fixing this 🌟

Reply:
Yeah. I'm done with reviewing the code. Right now I'm going through all the examples, new and old side-by-side, for a visual inspection.

Reply:
coolz!

Reply:
The reason colors messed up was that I picked whatever color name was there (which were lime brand color names) and used its equivalent variable name instead. But a bit inconsistently.

Anyhow, I pushed a fixup now, and they look like this 
![image](https://user-images.githubusercontent.com/35954987/114562794-1efafc00-9c6f-11eb-8a62-99cab52fedc8.png)